internal/exec/stages/disks/raid.go

```diff
@@ -48,6 +49,36 @@ func (s stage) createRaids(config types.Config) error {
 	}
 
 	for _, md := range config.Storage.Raid {
+		devName := md.Name
+		if !strings.HasPrefix(devName, "/dev") {
+			devName = "/dev/md/" + md.Name
+		}
+		if cutil.IsTrue(md.Assemble) {
+			args := []string{
+				"--assemble",
+				"--run",
+				"--homehost", "any",
+				devName,
+			}
+
+			for _, dev := range md.Devices {
+				args = append(args, util.DeviceAlias(string(dev)))
+			}
+
+			if _, err := s.Logger.LogCmd(
+				exec.Command(distro.MdadmCmd(), args...),
```

Comment:
it might help readablity if we make this command a var

aka cmd:= exec.Command(distro.MdadmCmd(), args...)

Reply:
not super opinionated on this, just a passing thought.